README.md

```diff
@@ -42,5 +45,7 @@ Then run:
 ```
 bundle
 bundle exec rake db:create
+bundle exec rake db:migrate
+bundle exec rake slack:update
```

Comment:
I would assume you would want to run this multiple times ( once a day? or every time a new user is added? )